package.json

```diff
@@ -18,86 +18,69 @@
     "karma:dev": "cross-env NODE_ENV=test karma start test/karma.config.js",
     "karma": "npm run karma:dev -- --single-run",
+    "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "preversion": "npm test",
     "standard": "standard",
     "test": "run-p standard karma wdio",
     "version": "npm run build && git add -A dist",
     "wdio:test": "cross-env NODE_ENV=test wdio test/wdio.config.js",
     "wdio": "npm run build wdio:test && git checkout dist/"
   },
-  "husky": {
-    "hooks": {
-      "pre-push": "npm run build && node scripts/check-staged.js"
-    }
-  },
-  "dependencies": {
-    "preact": "^8.3.1"
-  },
   "devDependencies": {
-    "@babel/core": "^7.1.5",
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/plugin-proposal-decorators": "^7.1.2",
-    "@babel/plugin-transform-member-expression-literals": "^7.0.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.1.0",
-    "@babel/plugin-transform-property-literals": "^7.0.0",
-    "@babel/plugin-transform-react-jsx": "^7.0.0",
-    "@babel/preset-env": "^7.1.5",
-    "@babel/register": "^7.0.0",
-    "@wdio/cli": "^7.16.13",
-    "@wdio/local-runner": "^7.16.13",
-    "@wdio/mocha-framework": "^7.16.13",
-    "@wdio/sauce-service": "^7.16.13",
-    "@wdio/spec-reporter": "^7.16.13",
-    "@wdio/static-server-service": "^7.16.13",
-    "@wdio/sync": "^7.16.13",
-    "babel-eslint": "^10.0.1",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-istanbul": "^5.1.0",
-    "chai": "^4.2.0",
-    "chalk": "^2.4.1",
-    "copy-webpack-plugin": "^6.4.1",
-    "cross-env": "^5.2.0",
-    "csso-cli": "^3.0.0",
-    "dotenv": "^6.1.0",
-    "husky": "^1.1.3",
-    "karma": "^6.3.9",
-    "karma-chai": "^0.1.0",
-    "karma-chai-sinon": "^0.1.5",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-mocha": "^1.0.1",
+    "@babel/core": "^7.23.3",
+    "@babel/eslint-parser": "^7.23.3",
+    "@babel/plugin-transform-react-jsx": "^7.23.4",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/register": "^7.22.15",
+    "@wdio/cli": "^7.33.0",
+    "@wdio/local-runner": "^7.33.0",
+    "@wdio/mocha-framework": "^7.33.0",
+    "@wdio/sauce-service": "^7.33.0",
+    "@wdio/spec-reporter": "^7.33.0",
+    "@wdio/static-server-service": "^7.33.0",
+    "babel-loader": "^8.3.0",
+    "babel-plugin-istanbul": "^6.1.1",
+    "chai": "^4.3.10",
+    "chalk": "^5.3.0",
+    "core-js": "^3.33.3",
+    "cross-env": "^7.0.3",
+    "csso-cli": "^4.0.2",
+    "husky": "^8.0.3",
+    "karma": "^6.4.2",
+    "karma-chrome-launcher": "^3.2.0",
+    "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.7",
+    "karma-sourcemap-loader": "^0.4.0",
     "karma-webpack": "^4.0.2",
-    "mocha": "^5.2.0",
+    "mocha": "^10.2.0",
     "npm-run-all": "^4.1.5",
-    "puppeteer": "^13.3.1",
-    "replace-bundle-webpack-plugin": "^1.0.0",
-    "sinon": "^6.3.5",
```

Comment:
Ah, I was [using](https://github.com/alphagov/accessible-autocomplete/pull/611/commits/84909c65471014b3189f29a33c19101f8429c1af#diff-4a14e2d8854c9e6cdf908f096dc93ec20897e9cc747b1515125ecfe0ad38bfa8R241) `useFakeTimers` from sinon to test the debounce.

Reply:
(I can do that with `setTimeout` and `done()` at the cost of some milliseconds)

Reply:
Thanks @rgarner, yeah probably best to use real timers in browsers